packages/paypal-commerce-integration/src/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -144,6 +187,64 @@ export default class PayPalCommercePaymentStrategy implements PaymentStrategy {
         return false;
     }
 
+    private prepareVaultedInstrumentPaymentPayload(
+        methodId: string,
+        paypalOrderId: string,
+        paymentData: VaultedInstrument & HostedInstrument,
+    ): Payment {
+        const { instrumentId, shouldSetAsDefaultInstrument } = paymentData;
+
+        return {
+            methodId,
+            paymentData: {
+                formattedPayload: {
+                    vault_payment_instrument: null,
+                    set_as_default_stored_instrument: shouldSetAsDefaultInstrument || false,
```

Comment:
`shouldSetAsDefaultInstrument` param can be passed outside `formattedPayload`

```suggestion
            paymentData: {
                shouldSetAsDefaultInstrument,
                formattedPayload: {
                    vault_payment_instrument: null,
```

Reply:
Is it necessary to provide it in this way?

Reply:
its ok to keep it like this